tests/api/test_config_factory.py

```diff
@@ -44,19 +44,26 @@ def config_dir(
 
 
 @pytest.fixture
-async def mock_for_login(aiohttp_server: _TestServerFactory, token: str) -> _TestServer:
+async def mock_for_login(
+    aiohttp_server: _TestServerFactory,
+    token: str,
+    aiohttp_unused_port: Callable[[], int],
+) -> _TestServer:
+
+    callback_urls = [
+        f"http://127.0.0.1:{aiohttp_unused_port()}",
```

Comment:
Do repeated calls of `aiohttp_unused_port()` return different values every time?

Reply:
It seems so

Reply:
Seems this is not guaranteed. There is a 0.005% chance of getting non-unique values in three calls of `aiohttp_unused_port()`. Hope this will not break the tests.